port_ocean/clients/port/mixins/entities.py

```diff
@@ -57,6 +57,13 @@ async def upsert_entity(
                 f"entity: {entity.identifier} of "
                 f"blueprint: {entity.blueprint}"
             )
+            result = response.json()
+            if (
+                response.status_code == 404
+                and result.get("ok") is False
+                and result.get("error") == "not_found"
```

Comment:
can we have the `not_found` string in a const of "port api error messages" enum? class? file?